service/pkg/util/fqn.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+	"errors"
+	"regexp"
+)
+
+type ResourceMappingGroupFqn struct {
+	Namespace string
+	GroupName string
+}
+
+// todo: patterns stored in variables for future extensibility
+// - could expose the patterns if needed
+
+// todo: temporary -> current namespace pattern in create proto validator is not correct (e.g. "example.com" fails validation, but is valid)
```

Comment:
That issue actually got closed after discussion with @strantalis.  The regex IS actually correct because it uses a slightly different regex format: [re2 syntax](https://github.com/google/re2/wiki/Syntax).  Maybe it's worth adding a comment about the differences and how this uses a more general regex to parse the namespace?